.github/workflows/rebase-bot.yml

```diff
@@ -1,11 +1,11 @@
+name: Rebase Bot
 on:
   issue_comment:
     types: [created]
-name: Rebase Bot
 jobs:
   rebase:
-    name: Rebase
-    if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/rebase') && github.repository == ‘kubevirt/hyperconverged-cluster-operator’
+    name: Build Manifests and Rebase
+    if: (github.event.issue.pull_request != '') && (contains(github.event.comment.body, '/rebase')) && (github.repository == 'kubevirt/hyperconverged-cluster-operator')
```

Comment:
Shouldn't the whole if statement be in parenthesis? e.g.
```
if: ((github.event.issue.pull_request != '') && (contains(github.event.comment.body, '/rebase')) && (github.repository == 'kubevirt/hyperconverged-cluster-operator'))
```

Reply:
I checked it on my fork and it works like it's written.
https://github.com/orenc1/hyperconverged-cluster-operator/blob/master/.github/workflows/rebase-bot.yml#L8
https://github.com/orenc1/hyperconverged-cluster-operator/actions/runs/127581061